vars/buildPlugin.groovy

```diff
@@ -121,7 +177,7 @@ def call(Map params = [:]) {
                   }
                   mavenOptions += 'clean install'
                   try {
-                    infra.runMaven(mavenOptions, jdk, null, null, addToolEnv)
+                    infra.runMaven(mavenOptions, jdk, null, settingsFile, addToolEnv)
```

Comment:
We'll keep this explicit "-s" for now as there is some path issue with ACI agents (Damien is working on it)